config/forms/2021_2022.json

```diff
@@ -1172,85 +1187,252 @@
                 }
               }
             },
-            "tenant_ethnic_group": {
+            "person_1_ethnic_group": {
               "header": "",
               "description": "",
               "questions": {
-                "ethnic": {
+                "ethnic_group": {
```

Comment:
Do we need to change the question key? These are generally chosen specifically to match ADDs data dictionary (since the question name matches the DB field name)

Reply:
Oh I see, sorry.

The saved answer we want to export as `ethnic` now comes as a follow on question. :heavy_check_mark: